### DISCUSSION TITLE
How to remove outline from Input when focus?

### DISCUSSION THREAD

**Discussion by alamenai** [shadcn-ui/ui#4118] [General]
I have an Input and I want to remove the outline when I focus:

![Screenshot from 2024-06-28 17-46-55](https://github.com/shadcn-ui/ui/assets/20143684/3ad3093d-f664-4e51-b62f-72b595a1cb2d)

This is my current code :

```tsx

  <Input
              className='p-6 pl-12 pr-24 focus-visible:outline-transparent placeholder:text-white/40 absolute bg-black/50 backdrop-blur-3xl border-transparent outline-none left-0 right-0 rounded-full focus-visible:ring-transparent focus-visible:border-transparent focus:ring-transparent focus:border-transparent'
              placeholder='Wohin reisen Sie in Deutschland?'
              value={address}
              onChange={(e) => setAddress(e.target.value)}
            />
```

**Comment by codesfromshad** [comment]
Might want to provide a bit more context. I can't seem to get the white ring when I focus with your tw-classes.

**Comment by alamenai** [comment]
Hi @codesfromshad ,

Yes, using the provided code, as you see in the input when I focus I can still see white outline around the input while I don't want to see it.

**Comment by incpo** [comment]
focus-visible:outline-none

**Reply by alamenai** [reply]
It remains the same: 

```
 <Input
                className='p-6 pl-12 pr-24 focus-visible:border-none focus-visible:outline-none placeholder:text-white/40 absolute bg-black/50 backdrop-blur-3xl border-transparent outline-none left-0 right-0 rounded-full'
                placeholder='Wohin reisen Sie in Deutschland?'
                value={address}
                onChange={(e) => setAddress(e.target.value)}
              />
```

![image](https://github.com/user-attachments/assets/a1d134db-9d3d-490f-a33d-f9221d24a4c0)

**Comment by AyushShivhare79** [comment]
Use focus-visible:ring-0 @alamenai
